File: 프로그래머스/unrated/181921. 배열 만들기 2/배열 만들기 2.py
def solution(l, r):
    answer = []
    for i in range(l, r+1):
        if i == 0:
            continue
        if set(str(i)) <= {'0', '5'}:
            answer.append(i)
    if not answer:
        answer.append(-1)
            
    return answer

File: 프로그래머스/unrated/181921. 배열 만들기 2/배열 만들기 2.js
function solution(l, r) {
    const answer = [];

    for (let num = l; num <= r; num++) {
        if(num == 0){
            continue
        }
        const numStr = num.toString();

        if (/^[05]+$/.test(numStr)) {
            answer.push(num);
        }
    }

    return answer.length === 0 ? [-1] : answer;
}